kettle/BUILD.bazel

```diff
@@ -23,6 +23,13 @@ py_binary(
     ],
 )
 
+py_binary(
```

Comment:
Please include its dependencies.

There are `pip` rules for both `gcloud` and `influx`. The `pip_import` rules should make adding them pretty trivial.

https://github.com/bazelbuild/rules_python/blob/master/docs/python/pip.md#pip_import

Reply:
yeah, I did that.